lib/github-glue.ts

```diff
@@ -100,21 +101,52 @@ export class GitHubGlue {
      */
     public async addPRCc(pullRequestURL: string, cc: string):
         Promise<void> {
-        const id = cc.match(/(<.*>)/);
+        const id = cc.match(/<(.*)>/);
 
         if (!id || id[1].match(/gitster@pobox\.com/)) {
             return;
         }
 
+        const ccLower = id[1].toLowerCase();
         const url = GitGitGadget.parsePullRequestURL(pullRequestURL);
         const pr = await this.getPRInfo(url[0], url[2]);
-        const ccNow = pr.body.match(/\r?\n\r?\n(cc:.*)/is);
+        const bodyEnd = pr.body.split(/\r?\n\s*?\r?\n/).pop();
```

Comment:
How certain are we that there is always an empty line? (I.e. is it possible for `pr.body` to contain just a single `Cc:` line?)

Reply:
Split will always return a non-empty array, even if the `pr.body` is empty.  If there is no empty line, split will return one entry in the array with `pr.body`.